src/autorest-core/resources/default-configuration.md

```diff
@@ -123,7 +123,7 @@ try-require: ./readme.typescript.md
 
 ``` yaml $(typescript)
 use-extension:
-  "@microsoft.azure/autorest.typescript": "~2.0.12"
+  "@microsoft.azure/autorest.typescript": "^4.1.1"
```

Comment:
You sure you want `^` ? 

> The tilde ~ matches the most recent patch version (the third number) for the specified minor version (the second number).
> ~1.2.3 will match all 1.2.x versions but will hold off on 1.3.0.
>
> The caret ^ is more relaxed. It matches the most recent minor version (the second number) for the specified major version (the first number).
> ^1.2.3 will match any 1.x.x release including 1.3.0, but will hold off on 2.0.0.

Usually we use `~` so that new patches rolled out are automatically the 'latest', but new minor versions are not.